app/kubernetes/views/security-constraint/constaintController.js

```diff
@@ -0,0 +1,26 @@
+// import _ from 'lodash-es';
+import angular from 'angular';
+import { FeatureId } from 'Portainer/feature-flags/enums';
+
+angular.module('portainer.app').controller('KubernetesSecurityConstraintController', [
+  '$scope',
+  'EndpointProvider',
+  'EndpointService',
+  function ($scope, EndpointProvider, EndpointService) {
+    $scope.limitedFeaturePodSecurityPolicy = FeatureId.POD_SECURITY_POLICY_CONSTRAINT;
+    $scope.state = {
+      viewReady: false,
+      actionInProgress: false,
+    };
+
+    async function initView() {
+      const endpointID = EndpointProvider.endpointID();
```

Comment:
EndpointProvider is deprecated. 
More detail:  https://portainer.atlassian.net/wiki/spaces/TECH/pages/45023521/Style+Guide#More

Could you please check if you can use the endpoint directly ?